dstep/driver/Application.d

```diff
@@ -37,20 +38,24 @@ class Application : DStack.Application
 
     private
     {
-        Language language;
+        Options defaultOptions;
```

Comment:
Yes, that was the reason I used `blueprint`, I can change it back or to something else. `defaultOptions` isn't that bad too.